src/thunderbots/software/radio_communication/visitor/mrf_primitive_visitor.cpp

```diff
@@ -0,0 +1,92 @@
+#include "radio_communication/visitor/mrf_primitive_visitor.h"
+
+RadioPrimitive MRFPrimitiveVisitor::getSerializedRadioPacket()
+{
+    return r_prim;
+}
+
+void MRFPrimitiveVisitor::visit(const CatchPrimitive &catch_primitive)
+{
+    r_prim.prim_type   = PrimitiveType::CATCH;
+    r_prim.param_array = {catch_primitive.getVelocity(),
+                          catch_primitive.getDribblerSpeed(),
+                          catch_primitive.getMargin()};
+    r_prim.extra_bits  = 0;
+}
+
+void MRFPrimitiveVisitor::visit(const ChipPrimitive &chip_primitive)
+{
+    r_prim.prim_type   = PrimitiveType::SHOOT;
+    r_prim.param_array = {chip_primitive.getChipOrigin().x() * 1000,
```

Comment:
What is `1000` here? Could we make it a descriptively named variable instead? Same goes for the all the "magic numbers" here.